src/components/Typeahead/Typeahead/Typeahead.js

```diff
@@ -32,8 +32,8 @@ export const Typeahead = ({
     <section className={`typeahead ${className | ''}`}>
       <div className="m-btn-inside-input input-contains-label">
         <div
-          className="input-contains-label_before
-                            input-contains-label_before__search"
+          className="input-contains-label__before
+                            input-contains-label__before__search"
```

Comment:
This is a modifier and should be updated to `input-contains-label__before--search`